### PR TITLE
feat: add i18n support with English and Brazilian Portuguese translations

### PR DESCRIPTION
**Pull request by voomsuetam**
This pull request introduces internationalization support based on the browser's language settings. Currently, English and Brazilian Portuguese are available.

### Details

- Implemented internationalization (i18n) to dynamically adjust the interface language according to the user's browser settings.
- Added support for two languages: English (en-US) and Brazilian Portuguese (pt-BR).

Peek 2024-08-04 04-19

### CODE REVIEW

**Review by Daniel-Boll — APPROVED** [review]

File: extension/locales/pt_BR/messages.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    "message": "Estudante"
+  },
+  "occupationLawyer": {
+    "message": "Advogado"
+  },
+  "occupationDoctor": {
+    "message": "Médico"
+  },
+  "occupationCivilEngineer": {
+    "message": "Desenvolvedor Civil"
```
Comment (Luisnadachi) [inline extension/locales/pt_BR/messages.json:63]: Aqui não deveria ser Engenheiro Civil?

File: extension/locales/pt_BR/messages.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  },
+  "occupationFrontEndEngineer": {
+    "message": "Desenvolvedor Front-end"
+  },
+  "occupationSreEngineer": {
+    "message": "Desenvolvedor SRE"
```
Comment (Luisnadachi) [inline extension/locales/pt_BR/messages.json:69]: Aqui não seria melhor somente SRE?

File: extension/locales/pt_BR/messages.json (hunk 1 above)
Comment (voomsuetam) [reply extension/locales/pt_BR/messages.json]: Fixed 🫡

File: extension/locales/pt_BR/messages.json (hunk 2 above)
Comment (voomsuetam) [reply extension/locales/pt_BR/messages.json]: Fixed 🫡

**Review by DanielHe4rt — APPROVED** [review]
LGTM!!

**Pull request merged**